### PR TITLE
Added support of survfitms

### PR DESCRIPTION
**Pull request by yoursdearboy**
Greetings.

This PR adds support of survfitms - multistate survival models built using `Surv(..., type = 'mstate')`.

Here is an overview of changes to `fortify`:

* To preserve original variable name depending on model class (`surv` or `pstate`) it is added to the end of the output data.frame;
* Where necessary checks for variable name are made;
* If `surv.connect = T` new rows are repeated according to number of states;
* In the end columns are sorted to be in usual order.

To `autoplot`:

* Added `facet_grid` support - argument `grid`. It's more convenient for 2-dim set plots;
* Added support to swap order of faceting (if facets) or column with rows (if grid). By default event first, then strata. Argument `strip_swap`;
* An argument `conf.int.group = NULL` was added to `plot_confint`. It's required to split CI ribbons by both - event and strata, because it can't be done with `fill` only;
* such *artificial* column `group` is assembled in `autoplot`.

Feedback or questions are welcome.

### CODE REVIEW

**Comment by sinhrks** [comment]
Thanks for the PR. Could you fix the Travis failure?

**Comment by yoursdearboy** [comment]
@sinhrks Everything should be ok now. I don't understand why Travis build failed, there's 0 warnings and errors.

**Comment by sinhrks** [comment]
Thx! A warning is raised because of the doc issue.

File: R/fortify_surv.R [hunk 1]
```diff
@@ -114,7 +159,9 @@ autoplot.survfit <- function(object, fun = NULL,
                              conf.int.fill = '#000000', conf.int.alpha = 0.3,
                              censor = TRUE, censor.colour = NULL, censor.size = 3,
                              censor.alpha = NULL, censor.shape = '+',
-                             facets = FALSE, nrow = NULL, ncol = 1, scales = 'free_y',
+                             facets = FALSE, nrow = NULL, ncol = 1,
+                             grid = FALSE, strip_swap = FALSE,
```
Comment (sinhrks) [inline R/fortify_surv.R]: can u add docs for `grid` and `strip_swap`?

**Comment by sinhrks** [comment]
Thanks! I'll fix the doc issue.

**Pull request merged**